docs/docs/user-guide/appendix/releasenotes-fw.md

```diff
@@ -1,5 +1,33 @@
 # Release Notes
 
+## BioNeMo Framework v2.5
+
+### New Features
+
+* Adding the Evo2 model training workflow, including data preprocessing, pre-training, fine-tuning and inference
+
+### Updates & Improvements
+
+* Adding NVFlare to ESM2 fine-tuning
```

Comment:
We are not adding NVFlare to fine-tuning. That would be on their repo. What we did was enabled them to do so by fixing the upstream issues in NeMo and BioNeMo that would otherwise block their demo

Reply:
```suggestion
* Supporting/upgrading federated learning examples of BioNeMo in [NVFlare](https://github.com/NVIDIA/NVFlare/tree/2.6.0rc1/examples/advanced/bionemo)
```